core/src/main/resources/reference.conf

```diff
@@ -1394,7 +1394,7 @@ datastax-java-driver {
       # Required: yes
       # Modifiable at runtime: no
       # Overridable in a profile: no
-      ticks-per-wheel = 512
+      ticks-per-wheel = 2048
```

Comment:
This is basically a number of buckets within a hash table (that is inside of the `HashedWheelTImer`), right?

Reply:
Yes.
2048 seems like a good default to minimize collisions, considering that out of the box the driver uses a request timeout of 2000ms and no speculative executions.